webserver/preview.php

```diff
@@ -32,6 +32,9 @@ function exception_error_handler($errno, $errstr, $errfile, $errline ) {
         "verify_peer"=>false,
         "verify_peer_name"=>false,
     ),
+    'http'=>array(
+      'header'=>"User-Agent: Mozilla/5.0 (iPad; U; CPU OS 3_2 like Mac OS X; en-us) AppleWebKit/531.21.10 (KHTML, like Gecko) Version/4.0.4 Mobile/7B334b Safari/531.21.102011-10-16 20:23:10\r\n" // i.e. An iPad
```

Comment:
Any particular reason you used the iPad user-agent?

Let's use the Facebook Crawler user-agent instead, because ultimately that'll be the user agent fetching the URL in production for converting and ingesting the Instant Article.

That's the user-agent string:
 `facebookexternalhit/1.1`.

See more: https://developers.facebook.com/docs/sharing/webmasters/crawler#identify

Reply:
Thanks for reviewing and I've updated user-agents. Actually I originally choose iPad without any specific reason.